rockcraft/oci.py

```diff
@@ -389,6 +389,22 @@ def set_cmd(self, command: str | None = None) -> None:
         _config_image(image_path, cmd_params)
         emit.progress(f"CMD set to {opt_args}")
 
+    def set_path(self, base: str, build_base: str | None) -> None:
+        """Set the default PATH on the image (only for bare rocks)."""
+        if base != "bare":
+            emit.debug(f"Not setting a PATH on the image as base is {base!r}")
+            return
```

Comment:
I think we could make this inference a bit more sustainable, as it could break if one day we have other bases without a `PATH` or we redefine the meaning of `bare` and actually serve `bare` as an existing OCI artefact, with Pebble and other settings (like `PATH`) already inside.

What about getting the base's existing `env` from `{image_path}/config.json`?

Reply:
Sure. Please add it as a good-first-issue. I'd love for it to be done by new joiners.

Reply:
https://github.com/canonical/rockcraft/issues/729